frontend/src/modules/Tables/views/TableView.js

```diff
@@ -157,7 +163,7 @@ const TableView = () => {
   const client = useClient();
   const navigate = useNavigate();
   const [table, setTable] = useState({});
-  const [currentTab, setCurrentTab] = useState('preview');
+  const [currentTab, setCurrentTab] = useState(tabs[0].value);
```

Comment:
@noah-paige Actually this was a mistake on my side... When contributing back Mohit's changes I forgot to pick this up from our branch. This will now always default to the first tab on the UI.